packages/core/lib/logger.js

```diff
@@ -1,7 +1,9 @@
-
-
 let logger = {
```

Comment:
don't we already have another logger module that uses Winston?

Reply:
This file is not used anymore, removing from the repo. But I think we can improve the sdk logging and start using winston properly.

We can add a custom transport to winston and use that in the webview, so that it will push the debug logs to the rect native thread via json-rpc. For the sdk it will be transparent, we are just pusing to winston. Also that is helpful when using nodejs, since the sdk user can configure winston as needed.

Reply:
https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871